src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -362,8 +362,9 @@ private function setRealFormat(string $format)
         $this->formatLineCount = substr_count($this->format, "\n");
     }
 
-    private function setMaxSteps(int $max)
+    public function setMaxSteps(int $max)
     {
+        $this->format = null;
```

Comment:
is this a bug fix?

Reply:
Bug fix. Wasn't issue before though, because this method was being called on the start only. Please note that format explicitly set by user is saved in different property.

Reply:
got it
what about moving the method up with other public ones?

Reply:
will do